config/config.go

```diff
@@ -148,3 +150,27 @@ func isSupportedBenchmark(benchmark string) bool {
 	}
 	return false
 }
+
+// TODO: move config validation to libs? (aws/gcp)
+// TODO: why is "" a valid case?
+func isSupportedAccountType(c Config) error {
```

Comment:
1 - not sure why we're allowing `""` as a valid case
2 - i think we should move this bit so each cloud vendor lib validates its own config, instead of just validating a single field here. (note `account_type` needs to be accompnied by either `project_id` or `organization_id`)

Reply:
As it is, isn't one of these two always going to be `""`? Since either gcp or aws is going to be enabled.

Reply:
i reverted my changes and kept the original aws account type check as it is. the GCP one is done at the credentials file where it also checks for matching IDs. 

> As it is, isn't one of these two always going to be ""? Since either gcp or aws is going to be enabled.

i think so, yeah. which i think is another reason to move this validation logic to libs.